include/usb_communication.h

```diff
@@ -0,0 +1,17 @@
+#ifndef USB_COMMUNICATION_H
+#define USB_COMMUNICATION_H
+
+#include <stdint.h>
+
+typedef struct oca_usb_data_frame oca_usb_data_frame;
+struct oca_usb_data_frame
+{
+  char data[128];	//TBD: Depends on the max protobuf frame size
+  uint8_t dataSize;
```

Comment:
size_t

Reply:
uint8_t because it directly maps to the single size byte in the USB protocol